wdio.conf.js

```diff
@@ -18,11 +18,14 @@ exports.config = {
     //
     specs: [
         './test/smoke/storyPage.js',
-        './test/regression/createButton.js'
+        './test/regression/createButton.js',
+        './test/regression/fieldName.js'
     ],
     // Patterns to exclude.
     exclude: [
-        // 'path/to/excluded/files'
+        './test/smoke/storyPage.js',
```

Comment:
Remove the list of the files

Reply:
should I leave only  // 'path/to/excluded/files' ?